README.md

```diff
@@ -51,11 +59,12 @@ In the environment you with to update regulations, first run:
 $ cf env eregs
 ```
 It will print to console the environment variables for the current running instance of eregs.
-Use that console output to copy / paste the appropriate parameters here:
+Use that console output to edit the file `~/.netrc` on your local machine.
 
-```bash
-$ export HTTP_AUTH_USER=[pasted from cf env output]
-$ export HTTP_AUTH_PASSWORD=[pasted from cf env output]
+```
+machine fec-dev-eregs.app.cloud.gov # This should match the hostname from FEC_EREGS_API in the cf env output
+login [copy and paste the value of HTTP_AUTH_USER from cf env output]
+password [copy and paste the value of HTTP_AUTH_PASSWORD from cf env output]
```

Comment:
These are the lines to edit in `~/.netrc`?  Sorry, was a bit confused at first, I thought they were supposed to be other environment variables!

Reply:
Yes, these are the lines to edit. After the edit, the file would look like:
```
machine fec-dev-eregs.app.cloud.gov 
login foobar
password foobars-top-secret-password
```